microprofile.h

```diff
@@ -3649,7 +3649,9 @@ void MicroProfileDumpHtml(MicroProfileWriteCallback CB, void* Handle, int nMaxFr
 					{
 						//for meta, store the count + 3, which is the tick part
 						nLogType = 3 + MicroProfileLogGetTick(pLog->Log[k]);
+		MicroProfileGetMutex().unlock();

```

Comment:
This is definitely not thread-safe since pLog can be deallocated if the thread exits. It also doesn't seem related to the PR in question, and may carry a large cost.